components/footer/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import classNames from './footer.module.scss';
+
+const Footer = () => {
+  return (
+    <footer>
+	   <p className={classNames.infoRepo}>The contents of this website are deployed from this <a href="https://github.com/Real-Dev-Squad/website-members" target="_blank" rel="noopener noreferrer">open sourced repo</a></p>
+	</footer>
```

Comment:
@sumitd94 this is not formatted properly. Can you please make sure to run `npm run lint`, `npm run format` before pushing your code.

Reply:
Ideally, this is handled by pre-commit hooks but I don't know how this got missed.